### ISSUE TITLE
Translations of ActiveRecord models and attributes

### ISSUE THREAD

**Issue by LegendSzymon-P** [rubysherpas/forem#60]
Models names and attributesm such as Topic and Post, are not translated - for example in a new topic form, i have "Subject" and "Text" instead of translated into in my case Polish.

But even when I add such as translation to my config/locale/pl.yml:

```
pl:
  activerecord:
    attributes:
      topic:
        subject: Temat
        text: Tekst
```

I still have `Subject` and `Text` as labels in new topic form. Why?

**Comment by radar** [comment]
Does `simple_form` attempt to reference translations by default automatically? If not, then that is probably the problem. Could you submit a pull request that makes these fields use the translations?

Thanks!

**Comment by LegendSzymon-P** [comment]
Ok, i've find out how simple_form is looking for attributes names and I've added polish translations to them in this pull request: https://github.com/radar/forem/pull/65

But there is still one problem - on the new and edit forms of topic there is a text on a button which uses name of model (Create Topic and Update Topic). I've find out so far that this is not forem thing, but simple_form, and it is using default rails-i18n paths for it: `helpers.submit.create` and `helpers.submit.update`.

I've finally just added translations of these buttons for each model: `helpers.submit.model.action`, but still I don't know how to translate models names which are used in these buttons, even when I add `activerecord.models.model` translation, it still is loading english name of model to these buttons, I don't know why, maybe because forem models are in `Forem` module?
